.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,91 @@
+name: \U0001F41B Bug report
+description: Report a bug in dependabot-core to help us fix it
+labels: ["T: bug 🐞"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        For support on the GitHub-integrated Dependabot service, please contact [GitHub support](https://support.github.com/).
+        This issue-tracker is meant for issues related to Dependabot's updating logic, a good rule of thumb is that if you have questions about the _diff_ in a PR, it belongs here, otherwise the GitHub support team is best equipped to help you.
+
+        The more information you can provide, the easier it will be to reproduce the issue and find a fix.
+
+  - type: checkboxes
+    attributes:
+      label: Is there an existing issue for this?
+      description: Please search existing issues to avoid creating duplicates.
+      options:
+        - label: I have searched the existing issues
+          required: true
```

Comment:
We were not mentioning this before in the previous bug template, thanks for adding it! Also makes sense to me to make it required.

Reply:
Agreed